### PR TITLE
Refactored cloud fraction

### PR DESCRIPTION
**Pull request by martinjanssens**
Quick one to shake off the post-holiday rust :)

### CODE REVIEW

File: cloudmetrics/metrics/cloud_fraction.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        Cloud mask field.
+
+    Returns
+    -------
+    cf : float
+        Clod fraction.
+
+    """
+
+    return len(np.where(cloud_mask == 1)[0]) / cloud_mask.size
```
Comment (leifdenby) [inline cloudmetrics/metrics/cloud_fraction.py:23]: This is fine, but there is also `np.count_nonzero` :) https://numpy.org/doc/stable/reference/generated/numpy.count_nonzero.html

**Review by leifdenby — APPROVED** [review]
Looks great! You can change that minor one if you want, I don't mind :)

File: cloudmetrics/metrics/cloud_fraction.py (hunk 1 above)
Comment (martinjanssens) [reply cloudmetrics/metrics/cloud_fraction.py]: Done!

**Comment by leifdenby** [comment]
Just a tip for your next pull-request. You can do a *squash* merge when you merge in the pull-request. That way your individual commits that you do while working on the pull-request don't appear in the final commit log. All you will get is just a single commit for the pull-request :) Also, you didn't update the changelog, but nevermind. We can do that in a another pull-request

**Comment by martinjanssens** [comment]
Ai, apologies for doing this inexactly. Squash merges sound nice! I've not touched the changelog since we started this, I'll keep that up to date properly in the future.

**Pull request merged**